notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -51,7 +51,15 @@
    "metadata": {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    import urllib.request
</pre><p>I believe for now downloading it will not work from branch, there is no such file on main yet</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1382/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
will it work when PR will be merged into master?

Reply:
yes, it will work, just it may makes review process complicated (we need to take attention on this part and manually copy file on desired location until PR is not merged) and lead to some troubles on ci